app/modules/site_settings/models.py

```diff
@@ -47,6 +47,49 @@ class SiteSetting(db.Model, Timestamp):
         'email_service_password': {'type': str, 'public': False},
         'email_default_sender_email': {'type': str, 'public': False},
         'email_default_sender_name': {'type': str, 'public': False},
+        'email_header_image_url': {
```

Comment:
Defaults need to be non flukebook specific, but otherwise, yes very much of an improvement.

Reply:
I thought we just need an example so people know what is expected.  The default shouldn't be used obviously :sweat_smile:

Reply:
Understood. The danger with that is that if you leave defaults that are valid for one project, they might get inadertently used by another and that would cause problems to debug.

Reply:
I think we need to have a send test email API and / or an API to show the basic email template so people knows what it'll look like

Reply:
Oh I agree with that entirely. All I'm saying is that noneof these defaults should be specific to any platform and currently they are.

Reply:
Oh I agree with that entirely. All I'm saying is that noneof these defaults should be specific to any platform and currently they are.

Reply:
We have a generic Wildbook logo, but not one for Codex yet. Perhaps the Wild Me logo could be used, that's an existing asset.